server/modules/coverpy.py

```diff
@@ -0,0 +1,109 @@
+import os
+import requests
+
+class NoResultsException(Exception):
+    pass
+
+class Result:
```

Comment:
can you also add some descriptive comments about the class at the starting of the class

Reply:
This is the CoverPy module that I'm using directly, I don't know how exactly is it working.